pkg/internal/testutils/testutils.go

```diff
@@ -41,42 +46,73 @@ func GetTestsDirWithDepth(depth int) string {
 	return filepath.Join(res, testsDirName)
 }
 
-// GetDebugMsgWithTestNameAndFormat: testing helping func - writes debug message for good path tests
-func GetDebugMsgWithTestNameAndFormat(testName, format string) string {
-	return fmt.Sprintf("test: %q, output format: %q", testName, format)
+// ConnlistTestNameByTestArgs returns connlist test name and test's expected output file from some tests args
+func ConnlistTestNameByTestArgs(dirName, focusWorkload, format string) (testName, expectedOutputFileName string) {
+	namePrefix := dirName
+	if focusWorkload != "" {
+		namePrefix += focusWlAnnotation + strings.Replace(focusWorkload, "/", underscore, 1)
+	}
+	testName = namePrefix + formatStr + format
+	expectedOutputFileName = namePrefix + underscore + connlistExpectedOutputFilePartialName + format
+	return testName, expectedOutputFileName
+}
+
+// DiffTestNameByTestArgs returns diff test name and test's expected output file from some tests args
+func DiffTestNameByTestArgs(ref1, ref2, format string) (testName, expectedOutputFileName string) {
+	namePrefix := "diff_between_" + ref2 + "_and_" + ref1
+	testName = namePrefix
+	if format != "" {
+		testName += formatStr + format
+	}
+	expectedOutputFileName = namePrefix + dotSign + format
```

Comment:
if `format` can be empty, how can a correct `expectedOutputFileName` be returned? 
cannot we assume `format` is not empty, and be consistent with `connlist` test names ?

Reply:
The question is why is it used in `diff_test` with empty format? why there is no consistent use in `DiffTestNameByTestArgs` in all diff_tests? Can it be changed to be consistent?